WtWSMS/common/factions/00_factions_BP_overwrite.txt

```diff
@@ -313,6 +326,21 @@ liberty_faction = {
 		}
 	}
 
+	##### Changed from Vanilla vvvvv
+	can_character_create_ui = {
+		common_can_character_create_trigger = {
+			FACTION_TARGET = scope:target
+		}
+
+		custom_description = {
+			text = has_lowest_crown_authority
+			subject = scope:target
+			scope:target = { realm_law_supports_liberty_faction_trigger = yes }
+		}
+	}
+	##### Changed from Vanilla ^^^^^
```

Comment:
Is this related to the Italy faction war bug?

Reply:
No, that's just a change in Vanilla for 1.8.*